code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -55,7 +55,7 @@
 		S.close(clientMob)
 
 /obj/item/weapon/storage/proc/setupItemBackground(var/HUD_element/itemBackground, var/atom/item, var/itemCount)
-	itemBackground.setClickProc(.itemBackgroundClick)
+	itemBackground.setClickProc(UNLINT(.itemBackgroundClick))
```

Comment:
not a great way to deal with this but its fine as long as someone comes back later to refactor this.